app/AppKernel.php

```diff
@@ -67,9 +67,16 @@ public function registerBundles()
          * @see https://symfony.com/doc/2.8/configuration/external_parameters.html#environment-variables
          */
         if (extension_loaded('apc')) {
-            $_SERVER['CACHE_DRIVER'] = 'apc';
+            $_SERVER['CACHE__DRIVER'] = 'apc';
```

Comment:
You'll need to change the %end(CACHE_DRIVER)% in the config_prod.yml as well

Reply:
true